amy/settings.py

```diff
@@ -131,6 +131,7 @@
     'rest_framework',
     'api',
     'captcha',
+    # 'debug_toolbar',
```

Comment:
Yes, see https://github.com/pbanaszkiewicz/amy/commit/5e776de2f2418effee904f407b80917a4646e088 for details. I use `debug_toolbar` application quite often and I am just bored adding and removing this line. (Un)commenting is just easier.